### PR TITLE
chore: add webworker to not supported table 

### PR DESCRIPTION
**Pull request by broofa**
For #392

### CODE REVIEW

File: README_js.md [hunk 1]
```diff
@@ -333,6 +333,7 @@ API is not supported. This issue can be resolved by adding an appropriate polyfi
 | Environment  | Suggested Polyfill                                                                                  |
 | ------------ | --------------------------------------------------------------------------------------------------- |
 | React Native | [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme) |
+| WebWorker (Edge) | N/A (let us know if you find something) |
```
Comment (TrySound) [inline README_js.md]: Worth to mention only old edge have this problem

Comment (broofa) [reply README_js.md]: Sure... do we have data on which Edge versions do/don't support crypto in webworkers?  The only thing I've found is the [caniuse page](https://caniuse.com/#feat=cryptography), which doesn't go into detail on versions.

Comment (TrySound) [reply README_js.md]: It does. 18 and below is old edge. 79+ is chromium based
image

**Comment by ctavan** [comment]
@broofa adjusted from the table to section layout.

**Pull request merged**